src/vs/editor/contrib/folding/browser/foldingRanges.ts

```diff
@@ -100,6 +100,14 @@ export class FoldingRegions {
 		}
 	}
 
+	public getStartIndexes(): Uint32Array {
+		return this._startIndexes;
+	}
+
+	public getEndIndexes(): Uint32Array {
+		return this._endIndexes;
+	}
+
```

Comment:
I believe using `getStartLineNumber` and `getEndLineNumber` with `length` will be the same as those new methods